rtgui/filmnegative.cc

```diff
@@ -265,9 +241,10 @@ FilmNegative::FilmNegative() :
     pack_start(*greenExp, Gtk::PACK_SHRINK, 0);
     pack_start(*redRatio, Gtk::PACK_SHRINK, 0);
     pack_start(*blueRatio, Gtk::PACK_SHRINK, 0);
-    pack_start(*spotButton, Gtk::PACK_SHRINK, 0);
+    pack_start(picker, Gtk::PACK_SHRINK, 0);
+    //    pack_start(*spotButton, Gtk::PACK_SHRINK, 0);
```

Comment:
There's a couple of commented code throughout this pull request that can be removed.